src/DynatraceMetricExport.ts

```diff
@@ -0,0 +1,29 @@
+/*
+	Copyright 2022 Dynatrace LLC
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+import { ExporterConfig, ReaderConfig } from "./types";
+import { PeriodicExportingMetricReader } from "@opentelemetry/sdk-metrics-base";
+import { DynatraceMetricExporter } from "./DynatraceMetricExporter";
+
+export function configureDynatraceMetricExport(exporterConfig: ExporterConfig, readerConfig: ReaderConfig): PeriodicExportingMetricReader {
```

Comment:
Minor, but since this is the primary export from this library, could we add a JSDoc comment to this function to give a basic description of its intent? The type annotations do help indicate what is going on, but a comment would also help in cases where users are not using TypeScript.

Reply:
I added a comment - I am not sure how in-depth we should go, especially if it is used without TS.

Reply:
Ideally we'd essentially have a rendered version of the TypeScript types inside the readme or a linked markdown file to serve as documentation, but I think what's there now is good.

Reply:
Is this common practice? I've never seen this before, but then again I haven't looked at that many TS repos...

Reply:
Added an overview table